src/Mod/Draft/draftutils/make_utils.py

```diff
@@ -0,0 +1,160 @@
+# ***************************************************************************
+# *   (c) 2019 Eliud Cabrera Castillo <e.cabrera-castillo@tum.de>           *
+# *                                                                         *
+# *   This file is part of the FreeCAD CAx development system.              *
+# *                                                                         *
+# *   This program is free software; you can redistribute it and/or modify  *
+# *   it under the terms of the GNU Lesser General Public License (LGPL)    *
+# *   as published by the Free Software Foundation; either version 2 of     *
+# *   the License, or (at your option) any later version.                   *
+# *   for detail see the LICENCE text file.                                 *
+# *                                                                         *
+# *   FreeCAD is distributed in the hope that it will be useful,            *
+# *   but WITHOUT ANY WARRANTY; without even the implied warranty of        *
+# *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the         *
+# *   GNU Library General Public License for more details.                  *
+# *                                                                         *
+# *   You should have received a copy of the GNU Library General Public     *
+# *   License along with FreeCAD; if not, write to the Free Software        *
+# *   Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  *
+# *   USA                                                                   *
+# *                                                                         *
+# ***************************************************************************
+"""Provides utility functions for make_polarray and make_circulararray."""
+## @package make_utils
+# \ingroup DRAFT
+# \brief This module provides functions for make_polarray & make_circulararray.
+
+import FreeCAD as App
+import Part
+
+import draftutils.utils as utils
+
+from draftutils.messages import _msg, _err
+from draftutils.translate import _tr
+
+
+def make_polcirc_shared(doc, _name, center=App.Vector(0, 0, 0),
```

Comment:
You should not name the parameter `_name` with an underscore. Just `name` should be fine.